internal/connor/any.go

```diff
@@ -0,0 +1,105 @@
+package connor
+
+import (
+	"github.com/sourcenetwork/defradb/client"
+
+	"github.com/sourcenetwork/immutable"
+)
+
+// anyOp is an operator which allows the evaluation of
+// a number of conditions over a list of values
+// matching if any of them match.
+func anyOp(condition, data any) (bool, error) {
+	switch t := data.(type) {
+	case []string:
```

Comment:
todo: please use the same approach as with `all`

Reply:
done